### PR TITLE
Power switching

### PR DESCRIPTION
**Pull request by tenzincrouch**
Updates to release one from debugging.

### CODE REVIEW

File: fsw/public_inc/master_comms_bus_protocol.h [hunk 1]
```diff
@@ -54,8 +52,6 @@ typedef enum
     SET_HEATER_STATE_MID = 39,        // 0b00100111,
     SET_HEATER_STATE_ALL_MID = 53,    // 0b00110101,
     HEATER_TELEM_MID = 40,            // 0b00101000,
-    GET_PERIPHERAL_DATA_MID = 43,     // 0b00101011,
-    PERIPHERAL_DATA_MID = 48,         // 0b00110000
 } master_comms_bus_msg_id_t;
 
 typedef struct {
```
Comment (sakshamkhurana) [inline fsw/public_inc/master_comms_bus_protocol.h:57]: Padding missing since master_comms_bus_msg_id_t is 8 bits and remaining members are 16 bit aligned.

**Review by sakshamkhurana — APPROVED** [review]

File: fsw/public_inc/nss_msgs.h [hunk 2]
```diff
@@ -35,17 +35,18 @@ typedef struct {
...
-#define SET_NSS_PARAMS_PAYLOAD_LEN sizeof(set_nss_params_payload_t);
+#define SET_NSS_PARAMS_PAYLOAD_LEN sizeof(set_nss_params_payload_t)
 
 // NSS telemetry message
 typedef struct {
     uint8_t nss_data[89];   // data from sensor
     uint8_t status;   // status byte: bit 0 = no telemetry, TODO: add more flags
                       // here
     msp_health_payload_t msp_health;
+    uint16_t __padding;   // ensure 32 bit alignment for telemetry struct
```
Comment (KJinator) [inline fsw/public_inc/nss_msgs.h:46]: this padding is not needed.  this only needs to be 16 bit aligned

Comment (KJinator) [reply fsw/public_inc/nss_msgs.h:46]: this padding is needed, all telemetry is 32 bit aligned

File: fsw/public_inc/master_comms_bus_protocol.h (hunk 1 above)
Comment (tenzincrouch) [reply fsw/public_inc/master_comms_bus_protocol.h:57]: This is an enum, so we dont need to padd it?

**Pull request merged**